root.tf

```diff
@@ -19,3 +19,45 @@ module "github_preservica_config_repository" {
     SLACK_WEBHOOK      = data.aws_ssm_parameter.github_slack_webhook.value
   }
 }
+
+module "closure_expiration_event_lambda" {
+  source          = "git::https://github.com/nationalarchives/da-terraform-modules//github_repository_secrets"
+  repository_name = "nationalarchives/dp-closure-expiration-event-generation"
```

Comment:
Do we want to put `lambda` or `function` in the repo name somewhere for ease of browsing? I know TRE use `tre-fn-...` as their naming convention.